documentation/src/main/asciidoc/topics/con_cache_store_sql.adoc

```diff
@@ -1,17 +1,16 @@
 [id='sql-cache-store_{context}']
 = SQL cache stores
 
-SQL cache stores pre-load {brandname} caches from databases, with a supported dialect, through JDBC connections.
-
-You can create two types of SQL cache store:
+SQL cache stores let you load {brandname} caches from database tables.
+{brandname} offers two types of SQL cache store:
 
 Table:: {brandname} loads entries from a single database table.
-Query:: {brandname} uses SQL queries to load entries as well as perform insert, update, and delete operations.
+Query:: {brandname} uses SQL queries to load entries from multiple database tables, select data from individual columns in database table rows, or perform insert, update, and delete operations.
```

Comment:
```suggestion
Query:: {brandname} uses SQL queries to load entries from possibly multiple database tables including possibly sub columns therein and perform insert, update, and delete operations if desired.
```

Reply:
I've reworded this sentence a little but won't commit these exact changes for grammar and style. How does this sound @wburns ?

SQL query cache stores let you load caches from multiple database tables, including from sub-columns in database tables, and perform insert, update, and delete operations.